ci/run_tests.yaml

```diff
@@ -0,0 +1,35 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: '1.9-alpine'
+
+params:
+  AWS_DEFAULT_REGION: 'eu-west-1'
+  TEST_R53_ZONE_NAME: ''
+  TEST_ACM_ARN: ''
+  TF_VERSION: '0.11.7'
+
+inputs:
+- name: terraform-src
+  path: gopath/src/github.com/skyscrapers/terraform-vault
+
+run:
+  path: sh
+  args:
+    - -exc
+    - |
+      apk add --update unzip curl dep git
+
+      # Install terraform
+      curl -s -o "terraform.zip" "https://releases.hashicorp.com/terraform/${TF_VERSION}/terraform_${TF_VERSION}_linux_amd64.zip"
+      unzip terraform.zip
+      export PATH=$PATH:$PWD
+
+      export GOPATH=$PWD/gopath
+
+      cd gopath/src/github.com/skyscrapers/terraform-vault/test
+      dep ensure
+      go test -v -timeout 60m
```

Comment:
There is only one small thing to consider. If the test fails for any reason some resources might still be there and not cleaned-up so it might be a good idea to make sure that all the test resources have been destroyed

Reply:
I'll created an issue to create a test AWS account, where to run the tests: https://github.com/skyscrapers/engineering/issues/37

Ok for you to approve this PR?